### PR TITLE
Setup the backend and add some script tooling

### PR DESCRIPTION
**Pull request by chaosphere2112**
Adds a script to setup the environment for a developer, and a script to automagically reload the front/backend code after filesystem events.

### CODE REVIEW

**Comment by mattben** [comment]
'LGTM' 👍

File: README.md [hunk 1]
```diff
@@ -1,30 +1,15 @@
...
+# vCDAT
 
+## Developer Setup
 
-Tips:  
-* `$(npm bin)` points you to the bin folder in the node_modules/ directory allowing you easier access to commands from npm modules without global installation.  
-Example:
+1. Fork the repo
+2. Clone your fork (`git clone git@github.com:$USERNAME/vcdat`)
+3. Install homebrew
```
Comment (doutriaux1) [inline README.md]: yuk 😉

Comment (chaosphere2112) [inline README.md]: It was that or write an entire cmake build system for prepping the developer environment. I chose the lesser of two evils.

Comment (aashish24) [inline README.md]: > yuk

While conda is good for some stuff :smile: , it is not a magic spell that will work on everything. 

@chaosphere2112 does linuxbrew works here?

**Comment by chaosphere2112** [comment]
@aashish24 No notion. This is the setup_mac script, so someone who wants to setup on linux should probably write a setup_linux script. Won't be for users; we'll distribute a wheel that can be installed via conda, instead.

File: README.md (hunk 1 above)
Comment (doutriaux1) [inline README.md]: @aashish24 you're right it's no magic. It just works :wink:

**Pull request merged**